### ISSUE TITLE
MissingPluginException (MissingPluginException(No implementation found for method convertHtmlToPdf on channel flutter_html_to_pdf))

### ISSUE THREAD

**Issue by jacobpoldenzuhlke** [Afur/flutter_html_to_pdf#46]
I am trying to generate in a PDF using an HTML string and getting the following exception:

```
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: MissingPluginException(No implementation found for method convertHtmlToPdf on channel flutter_html_to_pdf)
#0      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension>
#1      FlutterHtmlToPdf._convertFromHtmlFilePath
package:flutter_html_to_pdf/flutter_html_to_pdf.dart:42
<asynchronous suspension>
#2      FlutterHtmlToPdf.convertFromHtmlContent
package:flutter_html_to_pdf/flutter_html_to_pdf.dart:15
<asynchronous suspension>
#3      _MyAppState.generateExampleDocument
package:flutter_html_to_pdf_example/main.dart:74
<asynchronous suspension>
```

I can reproduce this error by cloning this repo and trying to run the example project using iPad emulator. 

Inspecting the /ios folder I cannot see any .symlinks being created for the project, not sure if that is a clear red flag or not.
Flutter Doctor output below:

```
flutter doctor -v
[✓] Flutter (Channel stable, 2.0.4, on Mac OS X 10.15.7 19H524 darwin-x64, locale en-GB)
    • Flutter version 2.0.4 at /usr/local/Caskroom/flutter/1.22.4/flutter
    • Framework revision b1395592de (5 days ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/japo/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Users/japo/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7042882/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Users/japo/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7042882/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3.2)
    • IntelliJ at /Users/japo/Applications/JetBrains Toolbox/IntelliJ IDEA Ultimate.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.55.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (2 available)
    • iPad Air (4th generation) (mobile) • 1C503043-DBC7-41D2-926E-981132587659 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • Chrome (web)                       • chrome                               • web-javascript • Google Chrome 89.0.4389.114

• No issues found!
```

**Comment by jacobpoldenzuhlke** [comment]
Issue likely to be fixed with PR https://github.com/Afur/flutter_html_to_pdf/pull/43

**Comment by UjjwaRaijada** [comment]
> Issue likely to be fixed with PR #43

Is there any way you can fix the issue at your end? Thanks

**Comment by harshit3105** [comment]
I've also facing this same issue in the iOS devices.

```[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: MissingPluginException(No implementation found for method convertHtmlToPdf on channel flutter_html_to_pdf) 
#0 MethodChannel._invokeMethod package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension> 
#1 FlutterHtmlToPdf._convertFromHtmlFilePath package:flutter_html_to_pdf/flutter_html_to_pdf.dart:42
<asynchronous suspension> 
#2 FlutterHtmlToPdf.convertFromHtmlContent package:flutter_html_to_pdf/flutter_html_to_pdf.dart:15

**Comment by parth-ibl** [comment]
Me also facing this same issue in iOS devices.

```#0 MethodChannel._invokeMethod package:flutter/…/services/platform_channel.dart:156
<asynchronous suspension>

**Comment by Afur** [comment]
Fixed with #43 by @jacobpoldenzuhlke . Sorry for the late response guys 🙏

**Comment by jacobpoldenzuhlke** [comment]
@Afur thank you :)
